packages/store/src/state-token/state-token.ts

```diff
@@ -18,20 +18,7 @@ export class StateToken<T = void> {
     return this.name;
   }
 
-  /**
-   * @description
-   * Hide the ability to serialize a token.
-   */
   public toString(): string {
     return `StateToken[${this.name}]`;
   }
-
-  /**
-   * @description
-   * Expose a fake method that returns the type T
-   * so that TS did not ignore the generic parameter.
-   */
-  protected _$implicitType(model: T): T {
-    return model;
-  }
```

Comment:
Oh! can we remove this strange method now?!
Of so, then awesome!!!

Reply:
done

Reply:
Fantastic!